CMakeLists.txt

```diff
@@ -66,6 +71,37 @@ if(MSVC OR MSVC_IDE)
     check_msvc_arch()
 endif()
 
+###############################################################################
+# Config
+###############################################################################
+
+# Install path
+set(BIN_INSTALL_DIR bin/ CACHE PATH "Installation directory for binaries")
+set(INCLUDE_INSTALL_DIR include/ CACHE PATH "Installation directory for C++ headers")
+set(LIB_INSTALL_DIR lib/ CACHE PATH "Installation directory for libraries")
+set(DATA_INSTALL_DIR share/ CACHE PATH "Installation directory for data")
+set(DOC_INSTALL_DIR ${DOC_DIR} CACHE PATH "Installation directory for documentation")
+if(MSVC OR MSVC_IDE)
+    set(LICENSE_INSTALL_DIR share/${PROJECT_NAME} CACHE PATH "Installation directory for licenses")
```

Comment:
This can be joined. Both produce the same directory.

Reply:
Maybe check where are the directories where Fast RTPS installs the License.

Reply:
Done!